schema/deploy/shipping/views.sql

```diff
@@ -223,4 +225,15 @@ grant select
    on shipping.incidence_model_observation_v2
    to "incidence-modeler";
 
+
+create view shipping.observation_with_presence_absence_result as
```

Comment:
Happy to change the view name if there's something else more appropriate!

Reply:
I think it's a fine name. Better than mine! :-)

Should we version this? I don't have particular thoughts here one way or another. I don't know if you considered it or not...

Reply:
I considered versioning it, but it's confusing that it uses observation v2 and presence/absence v1...

Reply:
Ah, that makes sense and might be confusing. I guess I'd see its version as starting at v1 and staying there so long as it kept using observation v2 and p/a v1 and the same join criteria.

I'm fine leaving it unversioned though and we can see how it goes.

Reply:
I slightly prefer the consistency of versioning this view.

Reply:
@joverlee521 I'll leave this decision up to you?

Reply:
Latest push added versioning to the view. We're start at v1 😄